Header header
string[] gripper_names
geometry_msgs/Pose[] gripper_poses
geometry_msgs/Point[] object_configuration
float32 sim_time
